src/components/PivotTable/Aggregators/Aggregators.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react'
+
+import { optionsKnob } from '@storybook/addon-knobs'
+import { action } from '@storybook/addon-actions'
+import { useLocale } from '../../../i18n'
+import { KeyMap } from '../model/model-keyMap'
+import { Aggregators } from './Aggregators'
+import { Aggregator } from './model'
+import { getCountAggregator } from './utils'
+
+export default {
+  title: 'Components/PivotTable/Aggregators',
+}
+
+type Fruit = {
+  name: string
+  size: string
+}
+
+const keyMapping: KeyMap<Fruit> = new Map([
+  ['name', { keyName: 'Name' }],
+  ['size', { keyName: 'Size' }],
+])
+
+const numberKeyOptions = {
+  Empty: [],
+  Name: ['name'],
+  'Name and Size': ['name', 'size'],
+}
+
+export const Default = () => {
+  const locale = useLocale()
+
+  const initialAggregator = getCountAggregator(locale.aggregators)
```

Comment:
Can I use here in stories other aggregators?

Reply:
Yes, this is just to set the default one, in stories you can choose the others :smile: